src/EnergyPlus/UnitarySystem.cc

```diff
@@ -288,13 +288,15 @@ namespace UnitarySystems {
                             if (speedNum < (maxSpeeds + 1)) {
                                 auto &cobj = flowRatio.at("cooling_speed_supply_air_flow_ratio");
                                 thisDesignSpec.coolingVolFlowRatio[speedNum] =
-                                    (cobj.type_name() == "string" && Util::SameString(cobj.get<std::string>(), "Autosize")) ? -99999
-                                                                                                                            : cobj.get<Real64>();
+                                    (cobj.type() == nlohmann::detail::value_t::string && Util::SameString(cobj.get<std::string>(), "Autosize"))
+                                        ? DataSizing::AutoSize
+                                        : cobj.get<Real64>();
```

Comment:
Thanks @mjwitte. Not sure where I got the other pattern.

Reply:
Also, I feel like I should follow Mr. Lohmann and introduce an `aroth` namespace into EnergyPlus. That would be a power move.